stylesheets/Components/toast.css

```diff
@@ -0,0 +1,13 @@
+.toast-container{
+    position: relative;
+}
+.toast{
+    background-color: var(--st-darkest-grey);
+    color: var(--st-white);
+    max-width: 60%;
+    padding: 5px 20px 5px 20px;
```

Comment:
Use shorthand for this
padding : 5px 20px;

Reply:
yeah..okay ...will do that